Tests/XCUITests/FindInPageTests.swift

```diff
@@ -173,12 +173,11 @@ class FindInPageTests: BaseTestCase {
         // Find in page is correctly launched, bar with text pre-filled and
         // the buttons to find next and previous
         if !iPad() {
-            mozWaitForElementToExist(app.buttons["Forward"])
-            app.buttons["Forward"].firstMatch.tap()
-            mozWaitForElementToExist(app.buttons["Forward"])
-            app.buttons["Forward"].firstMatch.tap()
-            mozWaitForElementToExist(app.buttons["Forward"])
-            app.buttons["Forward"].firstMatch.tap()
+            while !app.collectionViews.menuItems["Search with Firefox"].exists {
```

Comment:
Don't we have a mozWaitForElementTo NOT exist or similar? I have not use this `while !` in the tests before so I'm not familiar with this solution to be sure it will work better

Reply:
Because we need to tap on Forward until "Search with Firefox" exists for tapping.